modules/openapi-generator/src/main/resources/swift5/api.mustache

```diff
@@ -172,7 +177,11 @@ extension {{projectName}}API {
                     promise(.failure(error))
                 }
             }
-        }.eraseToAnyPublisher()
+        }{{#useURLSession}}
```

Comment:
@DavidHorvath I think here we can remove the checks on the URLSession, because now is alamofire also supports cancellation

Reply:
fixed